bioimageio/core/utils.py

```diff
@@ -0,0 +1,17 @@
+from marshmallow import missing
+
+from bioimageio.spec.shared.io import Node
+from bioimageio.spec.shared.nodes import ImportedSource
+
+try:
+    from typing import get_args
+except ImportError:
+    from typing_extensions import get_args  # type: ignore
+
+
+def get_nn_instance(model_node: Node, **kwargs):
```

Comment:
For pytorch_script_models it would be nice if this function would actually load the weights already.

Reply:
moved to pytorch model adapter...